src/appComponents/twitterwall/singletweet.js

```diff
@@ -44,20 +44,12 @@ function tweet($timeout,$rootScope) {
         scope.changeTweet=changeTweet;
         scope.callchangetweet=callchangetweet;
         scope.even=false;
-        getObject();
-        scope.$on('array',function(args,message){
-            scope.array=message;
-        });
-        function getObject(){
-            scope.$watch('array',function(oldvalue,newvalue){
-            if(scope.array){
-            scope.selectedid=scope.array[1].id_str;
-            changeTweet();
-            }
-            })
+         
+        function callchangetweet(){
+                $timeout(changeTweet,8000);
         }
         function changeTweet(){
-            if(scope.tweetNumber!=90){
+            if(scope.tweetNumber!==90){
 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of '++'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2412486393/issues/source?bid=3233857&fileBranchId=3235518#l54)